src/test/java/org/folio/innreach/controller/InnReachTransactionControllerTest.java

```diff
@@ -688,6 +699,64 @@ void return200HttpCode_and_sendRequest_whenItemHoldTransactionCreated() {
     assertEquals(user.getId(), transaction.get().getHold().getFolioPatronId());
   }
 
+  @Test
+  @Sql(scripts = {
+    "classpath:db/central-server/pre-populate-central-server.sql",
+    "classpath:db/mtype-mapping/pre-populate-material-type-mapping.sql",
+    "classpath:db/central-patron-type-mapping/pre-populate-central-patron_type-mapping-table.sql"
+  })
+  void return200HttpCode_and_sendRequest_whenItemHoldTransactionCreated_chosenPickLocationFromTransaction() {
```

Comment:
I don't see a test for this condition:
![image](https://user-images.githubusercontent.com/42244720/170971675-5f2cd458-dda8-4c5a-ab06-5ab7fe1ea99c.png)

Reply:
Fixed